lore/models/base.py

```diff
@@ -310,8 +315,9 @@ def save(self, upload=False):
             self.fitting.save()
 
     @classmethod
-    def load(cls, fitting_id=None):
-        model = cls()
+    def load(cls, *args, **kwargs):
```

Comment:
How does this work with passed in kwargs, aren't they obliterated during unpickling?

Reply:
Yes they are. Feel like it would be easier to chat about this in person. But the short answer is that the params are not really relevant for a downloaded estimator. The only reason I'm actually getting params is to initialize the class. Ideally, we would introspect the class and set some dummy values. Again, not sure what the cleanest way to do this is. Happy to get more input here